vars/virtualenv.groovy

```diff
@@ -1,16 +1,8 @@
-def create(String python, String destDir) {
-  sh("virtualenv --python=${python} ${destDir}")
-}
-
-
-def installRequirements(String destDir) {
-  run(destDir, "pip install -r requirements.txt")
-}
+import com.ableton.VirtualEnv
 
 
-def run(String destDir, String command) {
-  sh("""
-    . ./${destDir}/bin/activate
-    ${command}
-  """)
+def create(script, String python) {
+  venv = new VirtualEnv(script)
+  venv.initialize(python)
```

Comment:
I think we should inline `initialize` here. Having it as a member function that could be called again feels dangerous.

Reply:
I wish we could, but calling `shell.sh` throws a weird Jenkins exception. I can try again and give you a full stacktrace if you want more details.

Reply:
What about `venv.script.sh(...)`?

Reply:
Ah, sorry, I misread your comment. The exception is thrown if we try calling `script.sh` from inside of the `VirtualEnv` ctor. Yeah, we could (and perhaps should) inline it here. Will fix.